### DISCUSSION TITLE
Are dbt-core 1.8.x and duckdb 1.x.y really unsupported?

### DISCUSSION THREAD

**Discussion by am-giordano** [duckdb/dbt-duckdb#484] [Q&A] [answered]
Hi!

I have been using this plugin for about a week in a pet project to learn dbt and didn't realize until now that the dbt-core and duckdb versions I have been using are apparently incompatible with dbt-duckdb 1.9.0, although didn't get any errors or unexpected behaviour and only saw this by accident when running `dbt --version`:

```
Core:
  - installed: 1.8.9
  - latest:    1.8.9 - Up to date!

Plugins:
  - duckdb: 1.9.0 - Not compatible!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation
```

I have been using dbt-core 1.8.9 and duckdb 1.1.2 and the README.md in this repository states that "the latest supported version targets dbt-core 1.7.x and duckdb version 0.10.x", although the setup.cfg specifies `dbt-core>=1.8.0` and `duckdb>=1.0.0` as install-requirements.

Is it then safe to use dbt-core 1.8.x and duckdb 1.x.y versions?

Apologies in advance if I got it all wrong.

Best regards,
Alexander

**Comment by jwills** [comment, marked as answer]
Ack that README is out of date, will fix it

I think it will make the `dbt --version` command happy if you use dbt-duckdb 1.8.4, which was the last version of the project that specifically targeted dbt-core 1.8.x; we have largely moved the upstream work to dbt-core 1.9.0 which we thought was going to come out ~ 1 month ago but seems to be delayed

**Reply by am-giordano** [reply]
Ok got it! Thanks :)
